rca/people/admin_forms.py

```diff
@@ -6,23 +7,45 @@ class StudentPageAdminForm(WagtailAdminPageForm):
 
     def __init__(self, *args, **kwargs):
         super(StudentPageAdminForm, self).__init__(*args, **kwargs)
+
+        user = kwargs.get("for_user")
+
         # Don't allow changing the user account or image collection once set as they are tied to permissions
         if (
             self.instance.student_user_image_collection
             and self.instance.student_user_account
         ):
-            readonly_fields = [
+            disabled_fields = [
                 "student_user_image_collection",
                 "student_user_account",
             ]
-            for f in readonly_fields:
+            for f in disabled_fields:
                 # Wrapped in a try except as when a student uses this form, these fields are hidden
                 # See rca.people.models.StudentPage.basic_content_panels and rca.people.utils.PerUserPageMixin
                 try:
                     self.fields[f].widget.attrs["disabled"] = True
                 except KeyError:
                     pass
 
+        if self.instance.title and self.instance.slug and not user.is_student():
```

Comment:
This will probably do for now, but I think `user` can be None (tested when previewing a student page as a student) meaning this will `user.is_student` check will throw an AttributeError

Might be worth adding in a condition to see if we can actually check the `user` too